service/history/workflow/metrics.go

```diff
@@ -131,3 +134,74 @@ func GetPerTaskQueueFamilyScope(
 		tags...,
 	)
 }
+
+type ActivityExecutionState int
```

Comment:
nit: Shall we call this `Status` instead of `State`, to be consistent with the definition of workflow status. WorkflowState means something else.